src/processing/mode/android/AndroidBuild.java

```diff
@@ -179,6 +181,8 @@ public File createProject() throws IOException, SketchException {
 //      InputStream input = PApplet.createInput(getCoreZipLocation());
 //      PApplet.saveStream(new File(libsFolder, "processing-core.jar"), input);
       Base.copyFile(coreZipFile, new File(libsFolder, "processing-core.jar"));
+      
+      copySupportV4(libsFolder);
```

Comment:
Why is `android-support-v4.jar` being added to the repo? Further, I don't think we have legal rights to distribute it.